public/content/wrapped-ether/index.md

```diff
@@ -0,0 +1,59 @@
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Supply     | The supply of ETH is managed by the Ethereum protocol. The [issuance](/roadmap/merge/issuance) of ETH is handled by Ethereum validators when processing transactions and creating blocks.                           | wETH is an ERC-20 token whose supply is managed by a smart contract. New units of wETH are issued by the contract after it receives ETH deposits from users, or units of wETH are burned when a user wishes to redeem wETH for ETH.                                                                                                                                        |
+| Ownership  | Ownership is managed by the Ethereum protocol through your account balance.  | Ownership of wETH managed by the wETH token smart contract, not the Ethereum protocol.                                                                                                                                         |
+| Gas        | Ether (ETH) is the accepted unit of payment for computation on the Ethereum network. Gas fees are denominated in gwei (a unit of Ether).                                                                                    | You cannot pay gas for a transaction using wETH tokens (except when using a third-party relayer service).                                                                                                                                                                                              |
+
+## Frequently Asked Questions
+ 
+<ExpandableCard title="Do you pay to wrap/unwrap ETH?" eventCategory="/wrapped-ether" eventName="clicked Do you pay to wrap/unwrap ETH?">
+
+You pay gas fees to wrap or unwrap ETH using the wETH contract.
+
+</ExpandableCard>
+
+<ExpandableCard title="Is wETH safe?" eventCategory="/wrapped-ether" eventName="clicked Is wETH safe?">
+
+Nevertheless, wETH is generally considered secure because it is based on a simple, battle-tested smart contract. The wETH contract has also beeen formally verified, which is the highest security standard for smart contracts on Ethereum.
```

Comment:
Nevertheless?

Reply:
that also caught my attention, i think we can drop it